teamB/simchaewon/app.js

```diff
@@ -0,0 +1,58 @@
+const http = require("http");
+const express =require("express");
+const cors =require("cors");
+const morgan =require("morgan");
+require("dotenv").config();
+
+
+const {DataSource} = require("typeorm");
+
+const weDataSource=new DataSource({
+    type:process.env.TYPEORM_CONNECTION,
+    host:process.env.TYPEORM_HOST,
+    port:process.env.TYPEORM_PORT,
+    username:process.env.TYPEORM_USERNAME,
+    password:process.env.TYPEORM_PASSWORD,
+    database:process.env.TYPEORM_DATABASE
+})
+
+weDataSource.initialize()
+    .then(()=>{
+        console.log("Data Source has been initialized!")
+    })
+    .catch((err)=>{
+        console.error("Error during Data Sources initializaton", err);
+        weDataSource.destroy();
+    })
+
+const app = express();
+app.use(cors());
+app.use(express.json());
+app.use(morgan("dev"));
+
+
+app.get("/ping", (req, res)=>{
+    res.status(200).json({"message" : "pong"});
+})
+
+app.post("/user/signup", async(req, res)=>{
+    const {name, userName, userMail, password}=req.body;
+    await weDataSource.query(
+        "INSERT INTO users (name, user_name, user_mail, password) VALUES (?,?,?,?)",
+        [name, userName, userMail, password]
+    );
```

Comment:
- 채원님 ! 해당 쿼리문의 indentation 을 맞추기 위해서 쿼리문을 백틱으로 사용하는 것을 권장합니다 !!
```suggestion
    await weDataSource.query(
        `INSERT INTO users 
            (name, user_name, user_mail, password)
         VALUES (?,?,?,?)`,
        [name, userName, userMail, password]
    );
```